### ISSUE TITLE
Triple Messaging

### ISSUE THREAD

**Issue by RyanTheMan** [EpicGamer1YT/ProjectCrossComm#24]
After I had signed in and added a classmate as a friend when I messaged him the messages I sent and received appeared three times in the chat.

**Comment by VoidXpulse** [comment]
Did you click 3 times?

**Comment by VoidXpulse** [comment]
Also, format your bug report please

**Comment by CAM-o-man** [comment]
I've no idea. Did your chat partner also recieve 3 messages?

**Comment by EpicGamer1YT** [comment]
No, I know this problem. It's the repetition of creation of listeners, possibly by multiple clicking

**Comment by connormcdermid** [comment]
OK, that makes sense.
